src/Kconfig

```diff
@@ -21,3 +21,5 @@ rsource "ipc/Kconfig"
 rsource "math/Kconfig"
 
 rsource "library_manager/Kconfig"
+
+rsource "debug/telemetry/Kconfig"
```

Comment:
This should be in the previous patch "telemetry: Add systick_info part of telemetry". No functional impact as this commit is the first user, but just noting.

Reply:
However you must make sure every commit can compile and run basic tests (not telemetry tests)

Reply:
Every commit here does compile